examples/gpu-example.yaml

```diff
@@ -6,32 +6,31 @@ metadata:
     controller-tools.k8s.io: "1.0"
   name: gpu-example
 spec:
-  studySpec:
-      name: gpu-example
-      owner: crd
-      optimizationtype: maximize
-      objectivevaluename: Validation-accuracy
-      optimizationgoal: 0.99
-      metricsnames:
-        - accuracy
-      parameterconfigs:
-        - name: --lr
-          parametertype: double
-          feasible:
-            min: "0.01"
-            max: "0.03"
-        - name: --num-layers
-          parametertype: int
-          feasible:
-            min: "2"
-            max: "3"
-        - name: --optimizer
-          parametertype: categorical
-          feasible:
-            list:
-            - sgd
-            - adam
-            - ftrl
+  studyName: gpu-example
```

Comment:
`studyName` seems redundant to me. Why not use `metadata.name` as the name?

Reply:
StudyJobName and StudyName are different. You can create multiple StudyJob for the same Study.
When you do so, the results of these StudyJobs are plotted in the same graph.